.babelrc

```diff
@@ -8,7 +8,9 @@
       "root": ["./src"],
       "alias": {
         "Task1": "./src/components/task1",
```

Comment:
Please remove dead code (from task 1) from the codebase.

Reply:
@loginov-rocks fixed